.github/workflows/docker-image-dashboard.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image Den Dashboard
```

Comment:
Looks like the file isn't formatted right

Reply:
It follows the same format as Den DnsMasq and the other workflows

Reply:
@ihor-sviziev Thanks for pointing it out. Fixed the Yaml spacing issue.